lib/internal/cluster/master.js

```diff
@@ -70,6 +79,15 @@ cluster.setupMaster = function(options) {
   assert(schedulingPolicy === SCHED_NONE || schedulingPolicy === SCHED_RR,
          `Bad cluster.schedulingPolicy: ${schedulingPolicy}`);
 
+  const hasDebugArg = process.execArgv.some((argv) => {
```

Comment:
It stand for a typo-that-slips-in-when-you-use-a-bad-IDE-and-have-to-finish-a-feature-two-days-before-the-release-deadline
Mea culpa 🤦‍♂️ 
Ref: https://github.com/nodejs/node/pull/13460